The security_inode_unlink() function was used in filter_vfs_unlink()
to avoid lock ordering problems from using vfs_unlink() directly.
This is kept only for compatibility reasons with older Lustre versions
so that it is possible to upgrade/downgrade Lustre on the same kernel.

Index: linux+rh+chaos/security/security.c
===================================================================
--- linux+rh+chaos.orig/security/security.c
+++ linux+rh+chaos/security/security.c
@@ -60,6 +60,7 @@ int __init security_init(void)
 
 	return 0;
 }
+EXPORT_SYMBOL(security_inode_unlink);
 
 /* Save user chosen LSM */
 static int __init choose_lsm(char *str)
